src/NuGet.Core/NuGet.Protocol/Resources/PackageSearchResourceV3.cs

```diff
@@ -1,24 +1,54 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
 using System.Linq;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
 using NuGet.Protocol.Core.Types;
+using NuGet.Protocol.Model;
 
 namespace NuGet.Protocol
 {
     public class PackageSearchResourceV3 : PackageSearchResource
     {
+        private readonly HttpSource _client;
+        private readonly Uri[] _searchEndpoints;
+
+#pragma warning disable CS0618 // Type or member is obsolete
         private readonly RawSearchResourceV3 _rawSearchResource;
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
We should use global supressions instead.

Reply:
Same here VS was not offering any option to suppress this into GlobalSuppressions file. It seems other soIution is have to suppress for whole this file.

Reply:
You cannot suppress compiler warnings using attributes.

Reply:
Resolved offline